dockerfiles/squid-proxy/latest/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM centos
+MAINTAINER openshift-qe
+
+ENV SQUID_CACHE_DIR=/var/spool/squid \
+    SQUID_LOG_DIR=/var/log/squid \
+    SQUID_USER=proxy
+
+RUN yum install -y squid
+RUN chmod -R 777 /var/spool/squid /var/log/squid /var/run /etc/squid/squid.conf
```

Comment:
Process would run as the `root` group so I'd suggest using mode `770` for **directories** we need to write to and `740` for **files**  that we only need to read. Other than that everything looks awesome. Thank you!

Reply:
How about directories could be 775, files could be 744? `0` for other users is not usual

Reply:
IIRC from when I was a sysadmin `/var/spool/squid`, `/var/log/squid` and `/etc/squid/squid.conf` are only accessible to root as they can contain sensitive info.  This can easily be checked in default permissions. But in fact even easier approach would be to do:
```
chgrp root /var/spool/squid /var/log/squid /var/run /etc/squid/squid.conf
chmod g+rwx /var/spool/squid /var/log/squid
chmod g+r /etc/squid/squid.conf
```